Engine/game/savegame_components.cpp

```diff
@@ -1092,7 +1092,7 @@ ComponentHandler ComponentHandlers[] =
     },
     {
         "Characters",
-        1,
+        2,
```

Comment:
Please give new save formats much higher numbers in case there will be change in 3.* branch

Reply:
You're right, I somehow missed the Overlays component upgrade...
Do we really need to keep aligned to ags3 savegames? How much higher, I was kinda counting on nobody ever daring increase formats in the 3 branch ever again.

Reply:
This is int32, so it could be anywhere, like 100. It's an arbitrary thing.

Even if you dont support loading old saves, and dont want to know their format numbers, I still believe it's a good thing to have distinction, because it's easy for us, and will keep any emergency cases covered.
3.5.1 for example is still not released, and it's not known when ags4 branch will be having proper releases (and how they will look like).